src/Sylius/Component/Channel/Model/ChannelTypeInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Channel\Model;
+
+interface ChannelTypeInterface
```

Comment:
IMO it should be a final class instead of interface:
```suggestion
final class ChannelTypes
```
and add also a private constructor, so that you can't initialize that
```
private function __construct()
{
}
```